content/docs/command-reference/repro.md

```diff
@@ -124,12 +127,12 @@ only execute the final stage.
 - `-i`, `--interactive` - ask for confirmation before reproducing each stage.
   The stage is only executed if the user types "y".
 
-- `-p`, `--pipeline` - reproduce the entire pipelines that the stage file
-  `targets` belong to. Use `dvc dag <target>` to show the parent pipeline of a
-  target stage.
+- `-p`, `--pipeline` - reproduce the entire pipelines that the stage `targets`
+  belong to. Use `dvc dag <target>` to show the parent pipeline of a target
+  stage.
```

Comment:
This may supports .dvc files as well, which can be the beginning of a pipeline. Please test it to confirm, and if so update to just:
```suggestion
- `-p`, `--pipeline` - reproduce the entire pipelines that the `targets`
  belong to. Use `dvc dag <target>` to show the parent pipeline of a target.
```
> p formatting needed too.

Reply:
@sarthakforwet I see you did update this, did you test to confirm first? May I ask what test you ran? Thanks

Reply:
Well, I ran the following test - First created a file named `text.txt` then tracked it with `dvc add` and finally built my rest of the pipeline by adding stages to dvc.yaml file.

![dvc](https://user-images.githubusercontent.com/46163196/87420582-2cf1f280-c5f3-11ea-89a4-21bdd28c9c5b.PNG)

`dvc.yaml` file -

![yaml_file](https://user-images.githubusercontent.com/46163196/87421627-df768500-c5f4-11ea-9b95-3f07c2649a45.PNG)

Directory structure - 

![dvc_tree](https://user-images.githubusercontent.com/46163196/87421424-87d81980-c5f4-11ea-90f7-8e0a17649f54.PNG)
  

  
After then I reproduced the complete pipeline with the last stage as target whose output is - 

![pipeline](https://user-images.githubusercontent.com/46163196/87422064-97a42d80-c5f5-11ea-9b1a-25b33ce20a54.PNG)

Reply:
OK. How does that check whether this option supports .dvc as targets? (not just stage names). That is the change applied here to the doc.

Reply:
* .dvc files

Reply:
\* .dvc files

Reply:
> How does that check whether this option supports .dvc files as targets?

Even if you write the `.dvc` filename there, this option would work fine. 

![image](https://user-images.githubusercontent.com/46163196/87750754-ec27f280-c819-11ea-9070-f5b35db80ee6.png)

Reply:
And actually I'm its kind of confusing with the output for `.dvc` file. As you said `.dvc` files are not stages but here we can see from the output that it is considering them as one. So don't you think the output for .dvc files needs to be something different?

Reply:
Sound good, thanks.

> Its kind of confusing with the output for .dvc file...

I agree. Can you please open an issue on the core DVC repo about this output? Please link to this discussion as context (I'll leave it unresolved for that purpose.)

Reply:
P/s also do you think we should also report the weird -R output you posted in https://github.com/iterative/dvc.org/pull/1572#issuecomment-658272251 when you provide `./` as the target? That output is also confusing! It repeats some "stages" but using different relative paths, it seems.

Reply:
Ah never mind about that last comment. I see you created an `x/` directory. I was confused by that.

Reply:
>  I see you created an x/ directory

Yeah! actually I created the stage names same in both `dvc.yaml` files to check if we can have do something like that.

Reply:
Were you able to open that issue @sarthakforwet ?

Reply:
Yeah @jorgeorpinel. I opened that issue [#4233](https://github.com/iterative/dvc/issues/4233) and also mentioned this discussion there.

Reply:
Great. Resolving this then